raft/raft.go

```diff
@@ -230,7 +230,7 @@ func newRaft(c *Config) *raft {
 		nodesStrs = append(nodesStrs, fmt.Sprintf("%x", n))
 	}
 
-	r.logger.Infof("newRaft %x [peers: [%s], term: %d, commit: %d, applied: %d, lastindex: %d, lastterm: %d]",
+	r.logger.Infof("newRaft %d [peers: [%s], term: %d, commit: %d, applied: %d, lastindex: %d, lastterm: %d]",
```

Comment:
actually we do want to print out ID in hex. Do you see anywhere we print out id in dec in raft pkg? Probably we need to fix the dec ones.